### PR TITLE
Setup CI for the project

### PR DESCRIPTION
**Pull request by norshtein**
Solve #49.

### CODE REVIEW

**Comment by norshtein** [comment]
Hi @zhongyi-zhang @leonwanghui , please have a review on this PR, once this PR is merged, we can start to merge other PRs.

**Comment by norshtein** [comment]
I have also enabled CI in my repo: https://github.com/norshtein/osb-checker. You can preview the effect of the CI [here](https://github.com/norshtein/osb-checker/pull/1)

File: ci/2.13/before_script.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd 2.13/mocks
```
Comment (leonwanghui) [inline ci/2.13/before_script.sh]: I‘m wondering if it's better to use relative path `cd ./mock`.

Comment (norshtein) [reply ci/2.13/before_script.sh]: The script will be run in the root folder of the project, so `2.13/mocks` means `./2.13/mocks`, that's [this](https://github.com/openservicebrokerapi/osb-checker/tree/master/2.13/mocks) folder.

Comment (leonwanghui) [reply ci/2.13/before_script.sh]: Got it

**Comment by norshtein** [comment]
Hi @mattmcneeney , seems CI is not working, could you please make this project use travis in https://travis-ci.org?

**Comment by mattmcneeney** [comment]
@norshtein I think it's now enabled. I'm going to close and reopen #53 to see if that triggers it.

**Comment by mattmcneeney** [comment]
It's working!

**Pull request merged**